src/uct/ib/base/ib_md.c

```diff
@@ -347,12 +347,14 @@ static ucs_status_t uct_ib_md_post_umr(uct_ib_md_t *md, struct ibv_mr *mr,
                                        off_t offset, struct ibv_mr **umr_p)
 {
 #if HAVE_EXP_UMR
-    struct ibv_exp_mem_region mem_reg;
+    struct ibv_exp_mem_region *mem_reg = NULL;
     struct ibv_exp_send_wr wr, *bad_wr;
     struct ibv_exp_create_mr_in mrin;
     ucs_status_t status;
     struct ibv_mr *umr;
     struct ibv_wc wc;
+    int i = 0, list_size = 1;
+    size_t reg_length = 0x80000000;
```

Comment:
is it because we don't have max KLM size?
if so let's define as constant is ib_device.h